packages/api-clients/open-api/bffApi.yml

```diff
@@ -15616,6 +15615,29 @@ components:
       required:
         - results
         - pagination
+    PurposeDelegation:
+      type: object
+      additionalProperties: false
+      required:
+        - id
+        - delegateId
+        - delegateName
+        - delegatorId
+        - delegatorName
+      properties:
+        id:
+          type: string
+          format: uuid
+        delegateId:
+          type: string
+          format: uuid
+        delegateName:
+          type: string
+        delegatorId:
+          type: string
+          format: uuid
+        delegatorName:
+          type: string
```

Comment:
Could make sense to return here the same thing stuff that we return here PR https://github.com/pagopa/interop-be-monorepo/pull/1352?

Maybe it's not necessary for the FE so for now it's ok to merge this PR as it is

Reply:
I was thinking the same, I would align with what we do in the agreement process after #1352 and return something like

```
            id:
              type: string
              format: uuid
            delegate:
              $ref: "#/components/schemas/CompactOrganization"
            delegator:
              $ref: "#/components/schemas/CompactOrganization"
```